relayer/relayPackets.go

```diff
@@ -112,10 +117,9 @@ func (rp *relayMsgRecvPacket) Timeout() uint64 {
 
 func (rp *relayMsgRecvPacket) FetchCommitResponse(src, dst *Chain, sh *SyncHeaders) (err error) {
 	var dstCommitRes *chanTypes.QueryPacketCommitmentResponse
-
 	// retry getting commit response until it succeeds
 	if err = retry.Do(func() error {
-		dstCommitRes, err = dst.QueryPacketCommitment(sh.GetHeader(dst.ChainID).Header.Height, rp.seq)
+		dstCommitRes, err = dst.QueryPacketCommitment(sh.GetHeader(dst.ChainID).Header.Height-1, rp.seq)
```

Comment:
I'm thinking of updating the sdk's `QueryTendermintProof` to do the `-1` so this isn't littered all over the relayer as well. I'd like to ensure that change doesn't break the code. Let me know when a good time to attempt this change would be

Reply:
Would love to see that change @colin-axner whenever it lands I'll add it to the relayer (all I will need to do is thake out a bunch of +1s!)

Reply:
I can open a pr for it. It would be good to test that it doesn't break the relayer before merging it into the SDK